modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/WebUrlResourceTest.java

```diff
@@ -185,16 +198,39 @@ protected Long testGetUserAccountWebUrlsPage_getUserAccountId() {
 
 	@Override
 	protected WebUrl testGetWebUrl_addWebUrl() throws Exception {
-		return _addWebUrl(
-			randomWebUrl(), Contact.class.getName(), _user.getContactId(),
-			ListTypeConstants.CONTACT_WEBSITE);
+		return _addWebUrl();
+	}
+
+	@Override
+	protected WebUrl testGetWebUrlByExternalReferenceCode_addWebUrl()
+		throws Exception {
+
+		return testGetWebUrl_addWebUrl();
 	}
 
 	@Override
 	protected WebUrl testGraphQLWebUrl_addWebUrl() throws Exception {
 		return testGetWebUrl_addWebUrl();
 	}
 
+	@Override
+	protected WebUrl testPatchWebUrl_addWebUrl() throws Exception {
+		return _addWebUrl();
+	}
+
+	@Override
+	protected WebUrl testPatchWebUrlByExternalReferenceCode_addWebUrl()
+		throws Exception {
+
+		return testPatchWebUrl_addWebUrl();
+	}
+
+	private WebUrl _addWebUrl() throws Exception {
+		return _addWebUrl(
+			randomWebUrl(), Contact.class.getName(), _user.getContactId(),
+			ListTypeConstants.CONTACT_WEBSITE);
+	}
+
```

Comment:
why do we need this? Can we put this 
```
return _addWebUrl(
			randomWebUrl(), Contact.class.getName(), _user.getContactId(),
			ListTypeConstants.CONTACT_WEBSITE);
```
directly where we need it?

Reply:
@smottal sure we can, I just thought that this is easier then repeating the same code a couple of times.